test/spec/controllers/tasksCtrlSpec.js

```diff
@@ -29,10 +34,36 @@ describe('Tasks Controller', function() {
     });
   });
 
+  describe('removeTask', function() {
+    var task;
+
+    beforeEach(function() {
+      sandbox.stub(window, 'confirm');
+      task = specHelper.newTodo();
+    });
+
+    it('asks user to confirm deletion', function() {
+      scope.removeTask(task);
+      expect(window.confirm).to.be.calledOnce;
+    });
+
+    it('does not remove task if not confirmed', function() {
+      window.confirm.returns(false);
+      scope.removeTask(task);
+      expect(User.user.ops.deleteTask).to.not.be.calledOnce;
```

Comment:
This would pass if for some reason `User.user.ops.deleteTask` were called _more_ than once. You want `.to.not.be.called`.